### DISCUSSION TITLE
Disable a timesheet record for admins

### DISCUSSION THREAD

**Discussion by saraiyakush** [kimai/kimai#4555] [General]
Hello,

Is there a way to disable a timesheet record? As an organisation's admin, I would like to add holidays for the employees and also disable those entries so they always appear on everyone's timesheet and they cannot modify/delete them.

**Comment by kevinpapst** [comment]
I'd suggest to use this plugin for holidays: https://www.kimai.org/store/controlling.html

But in general: 
-"Disabling" a single timesheet record usually means setting it to "exported", as normal users cannot edit exported timesheet
- on a global level, there is also a lockdown mode (see System > Settings) which allows to block editing configurable time ranges
